### ISSUE TITLE
[accessibility] - When a new session is opened, the save & update dialog is inaccessible to a screen reader

### ISSUE THREAD

**Issue by SeanTolstoyevski** [bitwarden/browser#1736]
## Describe the Bug

Unable to find **"save dialog"** or **"update dialog"** with screen readers when logging into a website that is not registered with Bitwarden
Likewise, there is no "update dialog".

## Steps To Reproduce

* If you want, **you can open NVDA** if you know how to use NVDA. It would be better.

1. Open MSEdge.
2. Click Bitwarden in Extensions and unlock the vault.
3. In a new tab, type in a URL not previously registered with Bitwarden.
4. Log in to the website.

## Expected Result

Screen readers should have access to the **"save dialog"**.
I don't want to be misunderstood, but when using Lastpass, a dialog was created at the bottom of the page. It was working great. Maybe an advice...
This is perhaps an ARIA problem.

## Actual Result

Screen readers cannot find the "save dialog" in the page.
Dialog opens somewhere, maybe a popup but I don't know how to get to it.

Usually I **click Bitwarden** from extensions area and then save from **Add login"**. This takes a lot of time and is impractical.

## Environment

- Operating system: Windows 10
- Browser: Microsoft Edge (based Chromium) 89.0.774.57
- Build Version 1.49.1

## Additional Context

I am using [NVDA](https://github.com/nvaccess/nvda). I don't think this is NVDA's problem.
I did not test in Firefox. But there seems to be a problem with all Chromium-based browsers.

Excuse me for bad English.
If more details are requested, I can send them.

**Comment by cscharf** [comment]
Thanks @SeanTolstoyevski , much appreciated, will let you know if we need any other detail.

**Comment by SeanTolstoyevski** [comment]
Hi again @cscharf ,

I'm not having issues with Edge Chromium right now. A frame appears at the bottom of the page.

If it has been updated I would like to say it worked successfully.

**Comment by cscharf** [comment]
Thanks @SeanTolstoyevski , we did do a release last week, so this may have already been addressed in a round-about-way, as there were several library updates. I'm going to close this for now, but if it persists or happens again please feel free to re-open it. Thanks for the update!
